tutorials/ner_double/README.md

```diff
@@ -68,17 +68,22 @@ code. In code it looks like this:
 import spacy
 
 nlp = spacy.load("en_core_web_md") # load the base model
-drug = spacy.load("en_ner_drugs") # load the drug model
+drug_nlp = spacy.load("en_ner_drugs") # load the drug model
+# give this component a copy of its own tok2vec
+drug_nlp.replace_listeners("tok2vec", "ner", ["model.tok2vec"])
 
 # now you can put the drug model before or after the other ner
+# This will print a W113 warning but it's OK because we replaced listeners
```

Comment:
The W113 warning isn't related to the listeners. It's related to the fact that `en_ner_drugs` doesn't have vectors in its vocab and `en_core_web_md` does. But as long as `en_ner_drugs` `ner` doesn't require vectors (`include_static_vectors = false`), it will work because it will just ignore the `en_core_web_md` vectors.

The problem is that there's no way to inspect whether a component references the vocab vectors, so all we can do is warn.